themes/qgrid/source/js/menu.js

```diff
@@ -1,5 +1,9 @@
 function toggleVisibility(e) {
-	e.stopImmediatePropagation();
+	var target = e.target || e.srcElement;
+	if (target.dataset.stopPropagate) {
```

Comment:
in ui u use data-stop-propagation and here stopPropagate, how does it work?)

Reply:
like data-attributes))) http://htmlbook.ru/blog/atribut-data